go/vt/schemadiff/diff_test.go

```diff
@@ -335,9 +344,9 @@ func TestDiffTables(t *testing.T) {
 			dq, dqerr := DiffCreateTablesQueries(env, ts.from, ts.to, hints)
 			d, err := DiffTables(env, fromCreateTable, toCreateTable, hints)
 			switch {
-			case ts.isError:
-				assert.Error(t, err)
-				assert.Error(t, dqerr)
+			case ts.expectError != "":
+				assert.ErrorContains(t, err, ts.expectError)
+				assert.ErrorContains(t, dqerr, ts.expectError)
```

Comment:
I think we can use `assert.EqualError(t, err, ts.expectError)` here, no?

Reply:
It's true, we could, but I'd like to keep it this way, because sometimes we like to aggregate errors so as to return as much information as possible to the user. Right now the particular test cases generate either singular errors, or "terminal" errors that are not aggregated, but potentially we could also see an aggregated error here.